src/components/SortControl.js

```diff
@@ -17,9 +17,10 @@ const SortControl = ({currentSortOptionInput, handleSelect}) => {
          <div class='sortContainer'>
            <p>Sort by</p>
            <select value={currentSortOption} onChange={sortMovies} data-testid='select'>
-           {sortOptions.map((sortOption) => (
-                 <option value={sortOption}
-                 >{sortOption}</option>
+           {Array.from(sortOptions.entries()).map(([key, value]) =>  (
```

Comment:
You can use just `map` over array `const sortOptions = ["Release Date", "Title"]`